js_modules/dagster-ui/packages/ui-core/src/assets/AssetEvents.tsx

```diff
@@ -121,10 +120,10 @@ export const AssetEvents = ({
   const def = definition ?? cachedDefinition;
 
   const hasFilter =
-    combinedParams.status !== MaterializationHistoryEventTypeSelector.ALL ||
+    combinedParams.statuses?.length !== 3 || // TODO - see if there's a better way to do this
```

Comment:
wasn't sure what the idiomatic way is to do list comparison. If combinedParams.statuses isn't `[AssetEventHistoryEventTypeSelector.MATERIALIZATION, AssetEventHistoryEventTypeSelector.OBSERVATION, AssetEventHistoryEventTypeSelector.FAILED_TO_MATERIALIZE]` then we know a filter is applied

Reply:
I know it's gross but I think it'd be best to create a file constant like 

```ALL_TYPES = [[AssetEventHistoryEventTypeSelector.MATERIALIZATION, AssetEventHistoryEventTypeSelector.OBSERVATION, AssetEventHistoryEventTypeSelector.FAILED_TO_MATERIALIZE]]
```

and then make this an `isEqual(combinedParams.statuses, ALL_TYPES)`

(Also not sure if the statuses are really optional, if they are, we should apply the default value here so the comparison still works)

My only worry with the "3" is that we'll add a fourth type one day and forget to update this code. The same is true of ALL_TYPES, but that seems a bit more likely to be caught!